nibabel/freesurfer/tests/test_io.py

```diff
@@ -53,19 +51,19 @@ def test_geometry():
     """Test IO of .surf"""
     surf_path = pjoin(data_path, "surf", "%s.%s" % ("lh", "inflated"))
     coords, faces = read_geometry(surf_path)
-    assert_equal(0, faces.min())
-    assert_equal(coords.shape[0], faces.max() + 1)
+    assert 0 == faces.min()
+    assert coords.shape[0] == faces.max() + 1
 
     surf_path = pjoin(data_path, "surf", "%s.%s" % ("lh", "sphere"))
     coords, faces, volume_info, create_stamp = read_geometry(
         surf_path, read_metadata=True, read_stamp=True)
 
-    assert_equal(0, faces.min())
-    assert_equal(coords.shape[0], faces.max() + 1)
-    assert_equal(9, len(volume_info))
-    assert_equal([2, 0, 20], volume_info['head'])
-    assert_equal(u'created by greve on Thu Jun  8 19:17:51 2006',
-                 create_stamp)
+    assert 0 == faces.min()
+    assert coords.shape[0] == (faces.max() + 1)
+    assert 9 == len(volume_info)
+   # assert np.array_equal([2, 0, 20],volume_info['head'])
+    np.testing.assert_array_equal([2, 0, 20],volume_info['head'])
+    assert create_stamp == ['created by greve on Thu Jun  8 19:17:51 2006'] # this creates assertion error - should we just remove it?
```

Comment:
It's just a string. I think removing the list brackets will do the trick.

```suggestion
    assert create_stamp == 'created by greve on Thu Jun  8 19:17:51 2006'
```